lib/phoenix/controller.ex

```diff
@@ -1364,16 +1340,9 @@ defmodule Phoenix.Controller do
   @doc """
   Clears all flash messages.
   """
-  def clear_flash(conn) do
-    persist_flash(conn, %{})
-  end
-
-  defp flash_key(binary) when is_binary(binary), do: binary
-  defp flash_key(atom) when is_atom(atom), do: Atom.to_string(atom)
+  defdelegate clear_flash(conn), to: Phoenix.Controller.Flash
 
-  defp persist_flash(conn, value) do
-    put_private(conn, :phoenix_flash, value)
-  end
+  defdelegate flash_key(binary_or_atom), to: Phoenix.Controller.Flash
```

Comment:
Why do we need to delegate the `flash_key` one? Could we just remove it from here?